ConsentViewController/Classes/WebView.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  WebView.swift
+//  ConsentViewController
+//
+//  Created by Vilas on 8/7/19.
+//
+
+import UIKit
+import WebKit
+import JavaScriptCore
+
+public protocol ConsentWebViewHandler {
```

Comment:
What if we define the protocol here and make it mandatory for our client's ViewController to implement it? Then they'd have to add call something like:
```swift
consentViewController.consentDelegate(self)
```
What you think?

Reply:
will discuss on this when you have time over slack/call

Reply:
we have our protocol methods implementation in ConsentViewController so we have to pass "self" of ConsentViewController not ViewControllers.
ViewController's "self" is required for "Webview" not for protocol.